.changeset/wise-masks-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+"remix": patch
```

Comment:
```suggestion
```

Reply:
@MichaelDeBoey I'd prefer we not use these sort of changes as a basis for a minor release. I see this as any file that doesn't require its own special loader should be loaded as a file, so any files that we don't currently map to that should be considered a bug. Of course adding loader rules in perpetuity isn't a good long-term solution, but that's a separate conversation.